cmd/ci-reporter/cmd/github.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-github/v34/github"
+)
+
+// This regex is getting used to identify sig lables on github issues
+var sigRegex = regexp.MustCompile(`sig/[a-zA-Z-]+`)
+
+var (
+	newColumn = GithubProjectBoardColumn{
+		ColumnTitle: "New/Not Yet Started",
+		ColumnID:    4212817,
+	}
+	underInvestigationColumn = GithubProjectBoardColumn{
+		ColumnTitle: "In flight",
+		ColumnID:    4212819,
+	}
+	observingColumn = GithubProjectBoardColumn{
+		ColumnTitle: "New/Not Yet Started",
+		ColumnID:    4212821,
+	}
+	resolvedColumn = GithubProjectBoardColumn{
+		ColumnTitle: "Resolved",
+		ColumnID:    6798858,
+	}
+)
+
+type (
+	// ColumnTitle title of a github project board column
+	ColumnTitle string
+	// ColumnID ID of a github project board column
+	ColumnID int64
+)
+
+// GithubProjectBoardColumn specifies a github project board column
+type GithubProjectBoardColumn struct {
+	ColumnTitle ColumnTitle
+	ColumnID    ColumnID
+}
```

Comment:
Can be simplified to 

```suggestion
type GithubProjectBoardColumn struct {
	ColumnTitle
	ColumnID   
}
```

Reply:
I like it more explicit, so current version (i think it easier to understand also for someone not so familiar with go)